apps/web/src/studio/components/workflows/preferences/WorkflowSubscriptionPreferences.tsx

```diff
@@ -47,7 +47,7 @@ export const WorkflowSubscriptionPreferences: FC<WorkflowSubscriptionPreferences
   updateWorkflowPreferences,
   arePreferencesDisabled,
 }) => {
-  const [isOverridingPreferences, setIsOverridingPreferences] = useState(false);
+  const [isOverridingPreferences, setIsOverridingPreferences] = useState(workflowUserPreferences !== null);
```

Comment:
Fix to ensure the initial state for preference override toggle is correct to ensure the useEffect condition that resets the form value isn't fired on initial render.